sdk/context.py

```
@@ -440,7 +543,17 @@ def run(data):
         return result
 
     def close(self):
-        self.close_async().result()
+        try:
+            self.close_async().result()
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)